foodplanner/lib/services/school_class_service.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:convert';
+import 'package:foodplanner/models/schoolClass.dart';
+import 'package:http/http.dart' as http;
+
+class SchoolClassService {
+  final String apiUrl;
+
+  SchoolClassService({required this.apiUrl});
+
+  Future<List<SchoolClass>> fetchAllClasses() async {
+    final response = await http.get(Uri.parse('$apiUrl/api/Classrooms/GetAll'));
+
+    if (response.statusCode == 200) {
+      List<dynamic> jsonResponse = jsonDecode(response.body);
+      List<SchoolClass> classes = SchoolClass.fromJsonList(jsonResponse);
+      return classes;
+    } else {
+      throw Exception('Kunne ikke hente Klasser');
+    }
+  }
```

Comment:
Vi skal sende JWT i autherization header. Da det er vel kun lærere der skulle kunne fetch dette

Reply:
Den her sker ved oprettetelse af børn, for forældre den bruges til at få en liste over alle klasser så barnet kan blive sat i en klasse